### DISCUSSION TITLE
VS Code not suggesting auto-imports for <Image/>, <Link/>, and <Head/> in JavaScript files, but works in TypeScript - how to resolve?

### DISCUSSION THREAD

**Discussion by code-harsh-ui** [community/community#117260] [Programming Help]
### Body

My VS Code is not suggesting auto-imports for <Image/>, <Link/>, and <Head/>. I have tried uninstalling VS Code and its extensions, but I am still not receiving suggestions for these tags. Please help me resolve this issue. Additionally, I have noticed that if the file type is TypeScript or if I change the file extension to .ts, the auto-imports work correctly. However, if the file type is JavaScript, the suggestions are not working.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Moinuddin-lgtm** [comment]
Response #1

**Comment by eranga-mohotty** [comment]
Have you tried using an extension that supports both JS asd TS such as [Auto Import - ES6 & TS](https://marketplace.visualstudio.com/items?itemName=moppitz.vscode-extension-auto-import) maybe ?

**Reply by code-harsh-ui** [reply]
Yes, I tried this extension called "Auto Import - ES6, TS, JSX, TSX" by Sergey Korenuk, but it's not working. I also restarted my VS Code.

**Reply by code-harsh-ui** [reply]
![image](https://github.com/community/community/assets/85839958/71fe9972-2e7b-484a-a16f-d0588214a374)

**Comment by giraffeAz** [comment]
Try Live Server Extension
